FlowCrypt/src/main/java/com/flowcrypt/email/api/email/EmailUtil.kt

```diff
@@ -150,7 +150,7 @@ class EmailUtil {
         TextUtils.isEmpty(email) -> ""
         email?.contains("@") == true -> email.substring(email.indexOf('@') + 1)
         else -> ""
-      }
+      }.lowercase()
     }
```

Comment:
An empty string may not be the best choice. Actually, how come email is already nullable? When would it ever be null?

On browser extension in these situations I throw (cannot parse domain from email).